src/flake8_to_ruff/converter.rs

```diff
@@ -24,6 +25,7 @@ use crate::warn_user;
 pub fn convert(
     config: &HashMap<String, HashMap<String, Option<String>>>,
     black: Option<&Black>,
+    isort: Option<&ISort>,
```

Comment:
Would you prefer if this was combined with the `black` argument above as a single argument? Maybe something like a `(Option<&Black>, Option<&ISort>)` tuple? Or maybe even a new struct:

Reply:
I think what you have here is totally fine! A struct would also be reasonable, but honestly this is simple and clear so not sure that'd even be preferable.

Reply:
Using tuples like that isn't really idiomatic in Rust. Introducing a struct with named fields would however probably be a good idea if we assume that more such config structs will be introduced in the future ... maybe `struct ToolConfigs { black: Option<&Black>, isort: Option<&ISort> }`?

Reply:
Alternatively, what about a new struct to hold "external" options:

```rust
pub struct ExternalOptions {
  pub black: Option<Black>,
  pub isort: Option<ISort>,
}
```

Then `convert` could take `Option<ExternalOptions>` instead of `Option<&Black>` and it would be easy to extend upon for future "external" tools.

Reply:
Yeah what @not-my-profile suggested is good too. Either way.

(I'm not sure what additional configuration options we'd respect in the future, though it's plausible that e.g. we could grab the target Python version.)

Reply:
Great, pushed code with `ToolConfigs` option here: https://github.com/charliermarsh/ruff/pull/2082/commits/37a87df1ca5d96122db130509d34501a97f04b43

Reply:
(`ExternalConfig` is probably a better name than `ToolConfigs`) ... I wouldn't name it `ExternalOptions` since it contains the `Option` type which could be a bit confusing^^

Reply:
Renamed here: https://github.com/charliermarsh/ruff/pull/2082/commits/aa15c6ad0361769938ae536cad5fb584a2054630